Author: Hans-Christoph Steiner <hans@eds.org>
Subject: Makefile for libcutils
Description:
 Caveats: Lots of Android utilities link against libcutils but do not use the
 "atomic" stuff.  The atomic stuff only works on a limited number of CPU
 architectures, so for now, just don't include atomic on those architectures.
Last-Update: 2016-09-18

--- a/libcutils/dlmalloc_stubs.c
+++ b/libcutils/dlmalloc_stubs.c
@@ -15,6 +15,7 @@
  */
 
 #include "log/log.h"
+#include "dlmalloc.h"
 
 #define UNUSED __attribute__((__unused__))
 
